src/blog/2019-census/index.md

```diff
@@ -0,0 +1,93 @@
+---
+title: The 2019 Reactiflux Census
+author: vcarl
+date: '2020-07-07'
+description: 'We recently ran a user survey with the aim of better understanding just who is using Reactiflux (or, at least, those who felt comfortable answering a voluntary survey with demographic questions). The results are in.'
+---
+
+# Reactiflux census 2019
+
+Towards the end of 2019, we ran a user survey with the aim of better understanding just who is using Reactiflux (or, at least, who felt comfortable answering a voluntary survey with demographic questions). The results are in.
+
+The results were actually in a while back, but we got caught up with implementing a number of changes to moderation processes and tools, and ultimately delayed this post by a number of months. Sorry! The upside is that we'll also be adding an official Code of Conduct shortly.
+
+## The questions
+
+All questions were optional, with the intent that if a question made a respondent uncomfortable, they could omit it without being excluded from the results.
+
+- What is your racial or ethnic identity? (multiple choice)
+- How old are you?
+- What is your gender identity?
+- Do you identify as transgender?
+- What is your sexual orientation?
+- Are you a person living with a disability?
+- Do you identify with any of the following religions? (multiple choice)
+- Do you feel well represented within the Reactiflux Community?
+- Have you felt marginalized or harassed while in the Reactiflux Community?
+- Is there anything you'd like to suggest or tell the Reactiflux staff anonymously? (free answer)
+
+We carefully crafted these questions with the aim of being inclusive, and attempted to follow similar patterns set by other demographic-type surveys. If there's a question here that feels inappropriately phrased, or that you think could be improved, we would love to hear from you however you feel comfortable. You can give us feedback in [the #reactiflux-admin channel](https://discord.gg/VXpaGH), DM one of the admins or mods in Discord, or DM [@reactiflux](https://twitter.com/reactiflux) on Twitter, or via our [anonymous contact form](/contact).
```

Comment:
Broken Discord invite

Reply:
This will just join the server generally tho, right?

Reply:
No, it's like a 404